engine/test/utils/run_sql_file

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
We can reuse test scripts later, are these files duplicated
- dm/tests/_utils/run_sql_file
- tests/integration_tests/_utils/run_sql_file